integrations/tensorflow/test/CMakeLists.txt

```
@@ -4,4 +4,11 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-add_subdirectory(tf_integration)
+
+if(${IREE_ENABLE_NEW_INTEGRATION_TESTS})
+  add_subdirectory(tf_integration)
+endif()
+
+if(${IREE_ENABLE_TFLITE_INTEGRATION_TESTS})
+add_subdirectory(tflite_integration)
+endif()
```

nit: consistent indentation

```suggestion
if(${IREE_ENABLE_NEW_INTEGRATION_TESTS})
  add_subdirectory(tf_integration)
endif()

if(${IREE_ENABLE_TFLITE_INTEGRATION_TESTS})
  add_subdirectory(tflite_integration)
endif()
```